RandomData.Api.Tests/NumberGenerators/NumberGeneratorTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using RandomData.Api.NumberGenerators.Dto;
+using RandomData.Api.NumberGenerators.Generators;
+using Xunit;
+
+namespace RandomData.Api.Tests.NumberGenerators
+{
+    public class NumberGeneratorTests
+    {
+        [Fact]
+        public void NumberGenerator_WithDefaultParameters_ReturnsPositiveNumber()
+        {
+            //arrange
+            var generator = new NumberGenerator();
+            var parameters = new NumberParameters();
+
+            //act
+            var result = generator.GetRandom(parameters);
+
+            //assert
+            Assert.True(result >= 0);
```

Comment:
We decided to use FluentValidation so it should be:
```cs
result.Should().BeGreaterOrEqualTo(0);
```
instead of 
```cs
Assert.True(result >= 0);
```